src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -773,9 +787,45 @@ private Bid updateBid(Bid bid, BidResponse bidResponse) {
             // copy bidResponse.bidid to openrtb_response.seatbid.bid.id
             bid.setId(bidResponse.getBidid());
         }
+
+        final Float cpmOverride = ObjectUtils.defaultIfNull(cpmOverrideFromImp(imp), cmpOverrideFromRequest);
+        if (cpmOverride != null) {
```

Comment:
In Go they set price if it is more than zero, but probably it is bc they don't have nill.

Reply:
ok, added comment `Unconditionally set price if coming from CPM override`.